lib/SILGen/SILGenFunction.h

```diff
@@ -680,6 +680,24 @@ class LLVM_LIBRARY_VISIBILITY SILGenFunction
   void emitClassMemberDestruction(ManagedValue selfValue, ClassDecl *cd,
                                   CleanupLocation cleanupLoc);
 
+  /// Generates code to destroy linearly recursive data structures, without
+  /// building up the call stack.
+  ///
+  /// Example:
```

Comment:
`Example:` -> `E.x.: In the following we want to deinit next without recursing into next.`